jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -59,6 +68,37 @@ static AbsoluteUnixPath getAppRootChecked(JibPluginConfiguration jibPluginConfig
     }
   }
 
+  /**
+   * Gets the value of the {@code <to><image>} parameter. If this parameter is null,
+   * "gcr.io/distroless/java/jetty" is returned for war packaging, "gcr.io/distroless/java" is
+   * returned otherwise.
+   *
+   * @param jibPluginConfiguration the Jib plugin configuration
+   * @return the base image value
+   */
+  static String getBaseImage(JibPluginConfiguration jibPluginConfiguration) {
+    String baseImage = jibPluginConfiguration.getBaseImage();
+    if (baseImage == null) {
+      baseImage =
+          isWarPackaging(jibPluginConfiguration)
+              ? "gcr.io/distroless/java/jetty"
+              : "gcr.io/distroless/java";
+    }
+    return baseImage;
+  }
+
+  /**
+   * Returns true if the maven packaging type is "war"
+   *
+   * @param jibPluginConfiguration the Jib plugin configuration
+   * @return true if the maven packaging type is "war"
+   */
+  private static boolean isWarPackaging(JibPluginConfiguration jibPluginConfiguration) {
+    return jibPluginConfiguration.getProject() != null
+        && jibPluginConfiguration.getProject().getPackaging() != null
+        && jibPluginConfiguration.getProject().getPackaging().equals("war");
```

Comment:
`"war".equals(...)` and you can remove one null check.

(I wonder if it is possible that `jibPluginConfiguration.getProject()` returns null? In which case?)

Reply:
`jibPluginConfiguration.getProject()` can be null only with unit tests if we forget to add a "mock".

Reply:
Okay, if so, I'd remove `jibPluginConfiguration.getProject() != null` and have all tests mock it correctly. We generally don't want test code to affect the prod code.

Reply:
Done